ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class PostCommand extends IronfishCommand {
+  static summary = 'Post a raw transaction'
+
+  static description = `Use this command to post a raw transaction.
+
+  The output is a finalized posted transaction. The transaction is also added to the wallet, and sent out to the network.`
```

Comment:
Post will not add to wallet and send to network. That logic will move to broadcast.

Reply:
We don't have the ability to remove that from the wallet, it's an eventual consistency property of our wallet. No matter what, once the transaction is added to the wallet the wallet will always try to broadcast it out in the event loop.

Reply:
It will not in Post API and wallet.post(), we have discussed this last friday with all the engs. The approach is to split that logic from post to broadcast, broadcast still belongs wallet. But currently post logic will be new post() + broadcast()

Reply:
Ok, so we'll need to avoid adding it to the wallet as well then if we want to avoid broadcasting.

So create() -> post() -> add()